app/src/handlers/census/events/GainExperienceEvent.ts

```diff
@@ -25,7 +25,8 @@ import {Loadout} from '../../../constants/loadout';
 export default class GainExperienceEvent {
     public readonly world: World;
     public readonly zone: Zone;
-    public readonly timestamp: number;
+    public readonly worldId: number;
```

Comment:
There's `world` and `worldId`, please use World as it uses the correct ENUM.

Reply:
@microwavekonijn @Hailot I'm sure you didn't miss this on purpose, but this is quite the error that was missed :P